src/CleanRegex/Internal/PatternStrings.php

```diff
@@ -19,9 +19,12 @@ class PatternStrings
     private $autoCapture;
     /** @var Flags */
     private $flags;
-    /** @var string[] */
+    /** @var (string|Pattern)[] TODO: Changed; confirm */
     private $patterns;
```

Comment:
Looking at the `predefinition` method, Pattern is also supported; but I didn't see any test covering this case.